### DISCUSSION TITLE
VR CAMERA

### DISCUSSION THREAD

**Discussion by BenjaminBBResident** [praydog/REFramework#363] [General Questions]
Is there a way to make the camera static but still have tracking during cutscenes AND interactions that force the camera to move? This mod is fantastic and would be flawless if the camera is never forced to move I can handle it but it is “uncomfortable”

**Comment by praydog** [comment]
No. Turn on Decoupled Camera Pitch if you can't handle the vertical movement.

**Reply by BenjaminBBResident** [reply]
Thank you for the reply, I already did that but was curious about making it static all together I can handle the motion but it’s just uncomfortable but absolutely excellent work on this mod!! I know it’s a lot of work so I don’t expect it to be perfect but it’s damn near native VR and it’s amazing nonetheless! I know it’s a lot of work and I wish I knew more about how to do this type of modding so I could help with the camera
